README.md

```diff
@@ -1,31 +1,177 @@
 
-## Build Setup
+See it in action [HERE](https://schema.repronim.org/ui/)
 
+<!-- TOC -->
+
+- [reproschema-ui](#reproschema-ui)
+  - [List of supported input types](#list-of-supported-input-types)
+    - [Text inputs](#text-inputs)
+    - [Numerical values](#numerical-values)
+    - [Audio inputs](#audio-inputs)
+    - [Time and dates inputs](#time-and-dates-inputs)
+    - [Multiple options inputs](#multiple-options-inputs)
+    - [Others](#others)
+  - [App architecture](#app-architecture)
+  - [Reference material](#reference-material)
+  - [Serve the app on your computer](#serve-the-app-on-your-computer)
+    - [Install the dependencies](#install-the-dependencies)
+    - [Serve the app locally](#serve-the-app-locally)
+    - [Build for production with minification](#build-for-production-with-minification)
+    - [Build for production and view the bundle analyzer report](#build-for-production-and-view-the-bundle-analyzer-report)
+    - [Run unit tests](#run-unit-tests)
+    - [Run e2e tests](#run-e2e-tests)
+    - [Run all tests](#run-all-tests)
+
+<!-- /TOC -->
+
+## List of supported input types
+
+The `inputType` is checked by `src/components/InputSelector/InputSelector.vue` which then calls the appropriate scrip from those listed in the folder `src/components/Inputs/`.
+
+### Text inputs
+
+-   text: `WebTextInput/TextInput.vue`
+-   multitext: `MultiTextInput/MultiTextInput.vue`
+-   email: `EmailInput/EmailInput.vue`
+
+### Numerical values
+
+-   number: `WebIntegerInput/IntegerInput.vue`
+-   float:  `WebFloatInput/FloatInput.vue`
+
+### Audio inputs
+
+-   audioCheck: `AudioCheck/AudioCheck.vue`
+-   audioRecord: `WebAudioRecord/Audio.vue`
+-   audioPassageRecord: `WebAudioRecord/Audio.vue`
+-   audioImageRecord: `WebAudioRecord/Audio.vue`
+-   audioRecordNumberTask: `WebAudioRecord/Audio.vue`
+-   audioAutoRecord: `AudioCheckRecord/AudioCheckRecord.vue`
+
+### Time and dates inputs
+
+-   date: `YearInput/YearInput.vue`
+-   year: `YearInput/YearInput.vue`
+-   timeRange: `TimeRange/TimeRange.vue`
+
+### Multiple options inputs
+
+-   radio: `WebRadio/Radio.vue`
+-   slider: `SliderInput/SliderInput.vue`
+-   select: `SelectInput/SelectInput.vue`
+-   selectLanguage: `SelectInput/SelectInput.vue`
+-   selectCountry: `SelectInput/SelectInput.vue`
+-   selectState: `SelectInput/SelectInput.vue`
+
+### Others
+
+-   documentUpload: `DocumentUpload/DocumentUpload.vue`
+
+<!-- MultiPart
+StudySign
+SaveData
+Static
+StaticReadOnly -->
+
+<!-- ```
+├── AudioCheck
+├── AudioCheckRecord
+├── DocumentUpload
+├── EmailInput
+├── MultiTextInput
+├── SaveData
+├── SelectInput
+├── SliderInput
+├── Static
+├── StaticReadOnly
+├── TimeRange
+├── Utils
+│   └── SelectImage
+├── WebAudioRecord
+├── WebFloatInput
+├── WebIntegerInput
+├── WebRadio
+├── WebTextInput
+└── YearInput
+``` -->
+
+## App architecture
+
+![app architecture](docs/img/app_architecture.jpeg)
```

Comment:
The image mentions a `ContextItem.vue` script but I could not locate it.

Reply:
That's an outdated one! It goes like this:
```
App.vue -> Survey.vue -> SurveyItem.vue -> 

if "ui !== 'multipart' && ui !== 'section'" --> InputSelector.vue --> ..
else-if "ui === 'multipart'" --> MultiPart.vue --> SurveyItem.vue --> ..
else-if "ui === 'section'" --> Section.vue --> SurveyItem.vue --> ..
```

Reply:
Cool will create a new "figure"